docs/config-file-ort-yml.md

```diff
@@ -95,7 +95,7 @@ excludes:
     comment: "Packages for testing only."
 ```
 
-The above example excludes all of the following scopes for all projects: `testAnnotationProcessor`,`testApi`,
+The above example excludes all the following scopes for all projects: `testAnnotationProcessor`,`testApi`,
```

Comment:
Commit message: Maybe better ".ort.yml documentation".

Reply:
Actually, I don't think it was wrong before. But I'm also fine with the new wording.